examples/fakeserver/README.md

```diff
@@ -13,4 +13,31 @@ integrating with the service.
 
 ## Running
 
-// TODO(rag): Add instructions for running with docker after pushing an image.
+Users of this repository can just use the `run` task.
```

Comment:
Could you sections "Run the fakeserver with Gradle" and "Run the fakeserver with Docker" to clearly say users only need to choose either of ways.

Reply:
Good point, added subsections